core/chains/evm/txmgr/evm_tx_store_test.go

```diff
@@ -1214,25 +1213,16 @@ func TestORM_LoadEthTxesAttempts(t *testing.T) {
 		etx := mustInsertConfirmedMissingReceiptEthTxWithLegacyAttempt(t, txStore, 3, 9, time.Now(), fromAddress)
 		etx.TxAttempts = []txmgr.TxAttempt{}
 
-		q := pg.NewQ(db, logger.Test(t), cfg.Database())
-
 		newAttempt := cltest.NewDynamicFeeEthTxAttempt(t, etx.ID)
 		var dbAttempt txmgr.DbEthTxAttempt
 		dbAttempt.FromTxAttempt(&newAttempt)
-		err := q.Transaction(func(tx pg.Queryer) error {
-			const insertEthTxAttemptSQL = `INSERT INTO evm.tx_attempts (eth_tx_id, gas_price, signed_raw_tx, hash, broadcast_before_block_num, state, created_at, chain_specific_gas_limit, tx_type, gas_tip_cap, gas_fee_cap) VALUES (
-				:eth_tx_id, :gas_price, :signed_raw_tx, :hash, :broadcast_before_block_num, :state, NOW(), :chain_specific_gas_limit, :tx_type, :gas_tip_cap, :gas_fee_cap
-				) RETURNING *`
-			_, err := tx.NamedExec(insertEthTxAttemptSQL, dbAttempt)
-			require.NoError(t, err)
-
-			err = txStore.LoadTxesAttempts(ctx, []*txmgr.Tx{&etx})
-			require.NoError(t, err)
-			assert.Len(t, etx.TxAttempts, 2)
-
-			return nil
-		})
+
+		err := txStore.InsertTxAttempt(ctx, &newAttempt)
 		require.NoError(t, err)
+
+		err = txStore.LoadTxesAttempts(ctx, []*txmgr.Tx{&etx})
+		require.NoError(t, err)
+		assert.Len(t, etx.TxAttempts, 2)
 		// also check after postgres transaction is committed
```

Comment:
The next 4 lines are exact duplicates of previous 3 lines.
I think now we can delete the redundant checks. Previously it was done in addition to the above pg tx, which you now removed